js/game.js

```diff
@@ -0,0 +1,253 @@
+import {Player} from './players.js';
+import {Board} from './board.js';
+
+function returnAccumulatedScores(data) {
+
+  let dataToAccumulate = [];
+
+  do {
+
+    let elementToCompare = data.shift();
+    elementToCompare.apperances = 0;
+    for (let i = 0; i < data.length; i++) {
+      if (elementToCompare.x === data[i].x && elementToCompare.y === data[i].y) {

```

Comment:
Możesz wcześniej zdestrukturyzować to `data[i]`, żeby zapis był czytelniejszy:

`const { x, y, score } = data[i]`